sdk/common/src/main/java/io/opentelemetry/sdk/internal/AttributeUtil.java

```diff
@@ -107,7 +105,9 @@ public static Object applyAttributeLengthLimit(Object value, int lengthLimit) {
   }
 
   public static void addExceptionAttributes(
```

Comment:
One thing I've considered is retaining some sort of system property / env var to fallback to the built-in JDK exception rendering. Would offer a nice escape hatch to revert to the built-in jdk rendering in case there's any bugs in the code, but downside is it would be hard to know when it would be safe to finally remove.